modules/logging-operator-upgrading-all-ns.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+:_content-type: PROCEDURE
+[id="logging-operator-upgrading-all-ns_{context}"]
+= Upgrading the Cluster Logging Operator to watch all namespaces
+
+In logging 5.7 and older versions, the Cluster Logging Operator only watches the `openshift-logging` namespace.
+If you want the Cluster Logging Operator to watch all namespaces on your cluster, you must redeploy the Operator. You can complete the following procedure to redeploy the Operator without deleting your logging components.
```

Comment:
... in order to support multi-cluster log forwarding in other namespaces....

Some how incorporate to identify why we would do this

Reply:
Tried to incorporate this, lmk if it makes sense